compio-runtime/src/runtime/mod.rs

```diff
@@ -273,6 +296,19 @@ impl Runtime {
         })
     }
 
+    pub(crate) fn poll_task_with_flags<T: OpCode>(
```

Comment:
No need for this method. Just return flags in `poll_task`.

Reply:
done